Isosurfacing_3/include/CGAL/Isosurfacing_3/Zero_gradient.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022-2023 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+
+#ifndef CGAL_ISOSURFACING_3_ZERO_GRADIENT_H
+#define CGAL_ISOSURFACING_3_ZERO_GRADIENT_H
+
+#include <CGAL/license/Isosurfacing_3.h>
+
+#include <CGAL/Origin.h>
+
+namespace CGAL {
+namespace Isosurfacing {
+
+/**
+ * \ingroup IS_Domain_helpers_grp
+ *
+ * \brief Class template for a gradient that equates zero everywhere.
```

Comment:
```suggestion
 * \brief Class template for a gradient functor that returns `CGAL::NULL_VECTOR` everywhere.
```

Reply:
The API already says it returns `CGAL::NULL_VECTOR`